### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'rdata'

### ISSUE THREAD

**Issue by rithvikvibhu** [dnsviz/dnsviz#70]
This happens on `dnsviz==0.9.0`, but not on `0.8.2` so something has changed between these tags.

```
>>> graph.main(['graph', '-Thtml', '-O', '-r', '/tmp/probe_out.json'])                                                                                                 Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/rithvik/.pyenv/versions/HSEnv/lib/python3.7/site-packages/dnsviz/commands/graph.py", line 464, in main
    name_obj.populate_status(arghelper.trusted_keys, supported_algs=arghelper.args.algorithms, supported_digest_algs=arghelper.args.digest_algorithms, validate_prohibited_algs=arghelper.args.validate_prohibited_algs)
  File "/home/rithvik/.pyenv/versions/HSEnv/lib/python3.7/site-packages/dnsviz/analysis/offline.py", line 863, in populate_status
    self._populate_status(trusted_keys, supported_algs, supported_digest_algs, is_dlv, None, follow_mx)
  File "/home/rithvik/.pyenv/versions/HSEnv/lib/python3.7/site-packages/dnsviz/analysis/offline.py", line 837, in _populate_status
    self._populate_nxdomain_status(supported_algs)
  File "/home/rithvik/.pyenv/versions/HSEnv/lib/python3.7/site-packages/dnsviz/analysis/offline.py", line 2402, in _populate_nxdomain_status
    supported_algs)
  File "/home/rithvik/.pyenv/versions/HSEnv/lib/python3.7/site-packages/dnsviz/analysis/offline.py", line 2279, in _populate_negative_response_status
    self._populate_rrsig_status(query, soa_rrset_info, self.get_name(soa_owner_name), supported_algs, populate_response_errors=False)
  File "/home/rithvik/.pyenv/versions/HSEnv/lib/python3.7/site-packages/dnsviz/analysis/offline.py", line 1580, in _populate_rrsig_status
    rrsig_status = Status.RRSIGStatus(rrset_info, rrsig, None, zone_name, fmt.datetime_to_timestamp(self.analysis_end), supported_algs)
  File "/home/rithvik/.pyenv/versions/HSEnv/lib/python3.7/site-packages/dnsviz/analysis/status.py", line 230, in __init__
    if self.dnskey.rdata.algorithm in DNSKEY_ALGS_VALIDATION_PROHIBITED:
AttributeError: 'NoneType' object has no attribute 'rdata'
```

Trying to trace the code, I think it's this commit (lines selected): https://github.com/dnsviz/dnsviz/commit/992baacead282d4f927cdc2ac56a2ba0005e8457#diff-1e33847313500796d7f40588ef54bc13df321fbae972af733d9da7bd00fbf76fR228-R229

If self.dnskey is None, (as passed by `dnsviz/analysis/offline.py, line 1580`), then a few lines later, `self.dnskey.rdata.algorithm` must not be accessed.

**Comment by cdeccio** [comment]
Good find!  I'll push out a fix soon and will then roll out 0.9.1 with that fix.

**Comment by rithvikvibhu** [comment]
Awesome, thanks!

**Comment by cdeccio** [comment]
Fixed in 178b9d7135bfb714ad26a073dd18c797c2ee1511
